test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/052_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_ON.lua

```diff
@@ -0,0 +1,173 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate has NO "certificate" and "ForceProtectedService"=OFF at .ini file
```

Comment:
@dboltovskyi, thank you for notice, done in 7a54ec959b9d1fe9d0aec72b60f86477a720cae1